### PR TITLE
[DEV APPROVED] Updating Readme for Bower changes

### PR DESCRIPTION
**Pull request by slaywell**
@davidtrussler here's the overview of the changes for https://github.com/moneyadviceservice/frontend/pull/1590

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,12 +3,24 @@ Yeast
...
+## Bower
+
+As of 7/11/2016 Yeast is registered with the Bower registry so we can use semantic versioning to reference it as a Bower package, rather than pulling straight from GitHub as we were earlier.
+
+When updating Yeast, update the version in bower.json, and also _create a Git tag_ with the same version:
+
+`git tag -a v1.0.0 -m "Initial tag for Bower"`
+
+This will make the version available in the Bower registry.
+
```
Comment (davidtrussler) [inline README.md]: Should we also add the necessary changes that need to be made in frontend and other repos that depend on Yeast here?

**Comment by rnalexander** [comment]
:+1:

**Comment by Tr4pSt3R** [comment]
beaú 👍

File: README.md (hunk 1 above)
Comment (slaywell) [inline README.md]: Yep, sounds good. I've added this now.

**Pull request merged**